src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -20,15 +16,12 @@
 import teammates.common.exception.TeammatesException;
 import teammates.common.util.Assumption;
 import teammates.common.util.Const;
-import teammates.common.util.EmailType;
 import teammates.common.util.JsonUtils;
 import teammates.common.util.Logger;
-import teammates.common.util.SanitizationHelper;
 import teammates.common.util.StatusMessage;
 import teammates.common.util.StatusMessageColor;
 import teammates.common.util.Templates;
 import teammates.common.util.Templates.FeedbackSessionTemplates;
-import teammates.common.util.TimeHelper;
 import teammates.ui.pagedata.InstructorFeedbacksPageData;
 
 public class InstructorFeedbackAddAction extends InstructorFeedbacksPageAction {
```

Comment:
Currently `InstructorFeedbackAbstractAction` is extended by   `InstructorFeedbacksPageAction` and `InstructorFeedbackEditSaveAction`

And `InstructorFeedbackAddAction` extends `InstructorFeedbacksPageAction` 

What you were referring is to `InstructorFeedbackAddAction` extends directly `InstructorFeedbackAbstractAction` and move `InstructorFeedbacksPageAction` methods to abstract class 
Is that what you were saying :sweat_smile: